packages/loopring_v3/test/testExchangeAgents.ts

```diff
@@ -1,286 +0,0 @@
-import { BalanceSnapshot, ExchangeTestUtil } from "./testExchangeUtil";
-
-contract("Exchange", (accounts: string[]) => {
-  let ctx: ExchangeTestUtil;
-  let exchange: any;
-  let exchangeOwner: string;
-
-  let ownerA: string;
-  let ownerB: string;
-  let ownerC: string;
-  let ownerD: string;
-
-  const createExchange = async (bSetupTestState: boolean = true) => {
-    await ctx.createExchange(
-      ctx.testContext.stateOwners[0],
-      bSetupTestState
-    );
-    exchange = ctx.exchange;
-    exchangeOwner = ctx.exchangeOwner;
-  };
-
-  const authorizeAgentsChecked = async (
-    owner: string,
-    agents: string[],
-    authorized: boolean[],
-    from: string
-  ) => {
-    await exchange.authorizeAgents(owner, agents, authorized, { from });
-    const events = await ctx.assertEventsEmitted(
-      exchange,
-      "AgentAuthorized",
-      agents.length
-    );
-    for (const [i, event] of events.entries()) {
-      assert.equal(event.owner, owner, "owner unexpected");
-      assert.equal(event.agent, agents[i], "agent unexpected");
-      assert.equal(event.authorized, authorized[i], "authorized unexpected");
-
-      const isAgent = await exchange.isAgent(owner, agents[i]);
-      assert.equal(isAgent, authorized[i], "isAgent unexpected");
-    }
-  };
-
-  const whitelistAgentsChecked = async (
-    agents: string[],
-    authorized: boolean[],
-    from: string
-  ) => {
-    await exchange.whitelistAgents(agents, authorized, { from });
-    const events = await ctx.assertEventsEmitted(
-      exchange,
-      "AgentWhitelisted",
-      agents.length
-    );
-    for (const [i, event] of events.entries()) {
-      assert.equal(event.agent, agents[i], "agent unexpected");
-      assert.equal(event.authorized, authorized[i], "authorized unexpected");
-
-      for (const owner of [ownerA, ownerB, ownerC, ownerD]) {
-        const isAgent = await exchange.isAgent(owner, agents[i]);
-        assert.equal(isAgent, authorized[i] || owner === agents[i], "isAgent unexpected");
-      }
-    }
-  };
-
-  const onchainTransferFromChecked = async (
-    from: string,
-    to: string,
-    token: string,
-    amount: BN,
-    sender: string
-  ) => {
-    token = ctx.getTokenAddress(token);
-
-    // Simulate all transfers
-    const snapshot = new BalanceSnapshot(ctx);
-    await snapshot.transfer(from, to, token, amount, "from", "to");
-
-    // Do the transfer
-    await exchange.onchainTransferFrom(from, to, token, amount, {
-      from: sender
-    });
-
-    // Verify balances
-    await snapshot.verifyBalances();
-  };
-
-  before(async () => {
-    ctx = new ExchangeTestUtil();
-    await ctx.initialize(accounts);
-
-    ownerA = ctx.testContext.orderOwners[0];
-    ownerB = ctx.testContext.orderOwners[1];
-    ownerC = ctx.testContext.orderOwners[2];
-    ownerD = ctx.testContext.orderOwners[3];
-  });
-
-  after(async () => {
-    await ctx.stop();
-  });
-
-  describe("Agents", function() {
```

Comment:
These test will need some updating, but will still be needed. So will restore and update later if deleted now.

Reply:
I restored the file but comment out the current tests.